### ISSUE TITLE
训练loss为nan

### ISSUE THREAD

**Issue by paralleled** [Chaphlagical/Deblur-GS#24]
您好，我在进行训练时loss一直显示为NAN，对output中生成的文件进行render，得到的depth和render结果图片均为全黑，以下是终端内容：python train.py -s /root/Deblur-GS/Datasets/exblurf_dataset/exblur_release/dragon --eval --deblur --blur_sample_num 15 --bezier_order 5 (deblur_gs) 
Optimizing 
0 [05/04 07:51:55]
Output folder: ./output/ed43a00d-1 [05/04 07:51:55]
Tensorboard not available: not logging progress [05/04 07:51:55]
Setting up a new session...
Reading camera 23/23 [05/04 07:51:55]
Using cache found in /root/.cache/torch/hub/intel-isl_MiDaS_master
/root/micromamba/envs/deblur_gs/lib/python3.12/site-packages/timm/models/layers/__init__.py:48: FutureWarning: Importing from timm.models.layers is deprecated, please import via timm.layers
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.layers", FutureWarning)
/root/micromamba/envs/deblur_gs/lib/python3.12/site-packages/timm/models/_factory.py:126: UserWarning: Mapping deprecated model name vit_base_resnet50_384 to current vit_base_r50_s16_384.orig_in21k_ft_in1k.
  model = create_fn(
Loading Training Cameras [05/04 07:51:59]
Loading Test Cameras [05/04 07:52:02]
Number of points at initialisation :  5752 [05/04 07:52:03]
Training progress:   1%|█▍                                                                                                                          | 1000/90000 [05:38<8:16:29,  2.99it/s, Loss=nan]blur_weight=0.06666666666666667 [05/04 07:57:41]

[ITER 1000] Evaluating test: L1 0.39292244116465247 PSNR 7.931527137756348 SSIM 0.00043641691445373 LPIPS 0.7230097850163777 [05/04 07:57:46]
0 gaussian_trans=Parameter containing:
tensor([[nan, nan, nan, nan, nan, nan],
        [nan, nan, nan, nan, nan, nan]], device='cuda:0', requires_grad=True) [05/04 07:57:46]
1 gaussian_trans=Parameter containing:
tensor([[nan, nan, nan, nan, nan, nan],
        [nan, nan, nan, nan, nan, nan]], device='cuda:0', requires_grad=True) [05/04 07:57:48]
2 gaussian_trans=Parameter containing:
tensor([[nan, nan, nan, nan, nan, nan],
        [nan, nan, nan, nan, nan, nan]], device='cuda:0', requires_grad=True) [05/04 07:57:49]
3 gaussian_trans=Parameter containing:
tensor([[nan, nan, nan, nan, nan, nan],
        [nan, nan, nan, nan, nan, nan]], device='cuda:0', requires_grad=True) [05/04 07:57:51]
4 gaussian_trans=Parameter containing:
tensor([[nan, nan, nan, nan, nan, nan],
        [nan, nan, nan, nan, nan, nan]], device='cuda:0', requires_grad=True) [05/04 07:57:53]

[ITER 1000] Evaluating train: L1 0.39967164993286136 PSNR 7.789648246765137 SSIM 0.0004649806767702103 LPIPS 0.6938338994979859 [05/04 07:57:54]

[ITER 1000] Saving Gaussians [05/04 07:57:56]

**Comment by teacherlcy** [comment]
我遇到了同样的问题，请问您解决了吗
